definitions/npm/backbone_v1.3.x/flow_>=v0.25.x/backbone_v1.3.x.js.flow

```diff
@@ -0,0 +1,68 @@
+// @flow
+import typeof Underscore from 'underscore';
+
+declare module 'backbone' {
+   declare var $: any;
+   declare var _: Underscore;
+   declare var version: string;
+   declare var emulateHTTP: boolean;
+   declare var emulateJSON: boolean;
+
+   declare type eventCallback = (event: Event) => void;
```

Comment:
Should use `mixed` instead of `void` here. Backbone doesn't care about it, but it will allow reused functions with returns to work with Flow.